test/powershell/Language/Scripting/ErrorPosition.Tests.ps1

```diff
@@ -1,51 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The tests still seem relevant though -- they do cover an existing bug and after it's fixed would check for regressions. We could keep them with a `-Pending` flag and track it with an issue?

Reply:
The tests will be added back along with the next attempt to fix the issue.